### PR TITLE
select only inputs and preview inspector in #editor

### LINKED ISSUE #112: Moving the canvas on Firefox selects elements

### PR DESCRIPTION
**Pull request by anewuser**
fixes #112

### CODE REVIEW

**Comment by onli** [comment]
That's ready now, right?

**Comment by anewuser** [comment]
Yes, please test it out.

**Comment by onli** [comment]
Works perfectly fine for me. Thanks!

File: public/style.css [hunk 1]
```diff
@@ -3,6 +3,15 @@
...
+    user-select: none;
+}
+
+#editor input {
+    user-select: auto;
+}
+
+#editor #inspector {
+    user-select: text;
 }
```
Comment (onli) [inline public/style.css:15]: Note: I was considering whether that line should rather be added to the `#inspector` block below. But it's an undecidable style question,  and having it under `#editor` could avoid issues.

Comment (anewuser) [reply public/style.css:15]: Ah, I just placed everything at the top to make it easier to change the three rules at once/remember why they were added without having to add comments elsewhere. Feel free to edit anything the way you want.

**Pull request merged**